lib/ssl/src/ssl_connection.hrl

```diff
@@ -115,7 +116,7 @@
                                                            rsa_pss_pss => list(),
                                                            rsa => list(),
                                                            dsa => list()
-                                                          } | secret_printout() | 'undefined'
+                                                          } | list() | 'undefined'
```

Comment:
Because it is part of connection_env that will be replaced by secret_printout as a whole, and this value may internally be
[[]] if there is no client cert configured.